### ISSUE TITLE
js-xlsx updates ?

### ISSUE THREAD

**Issue by lbayle** [Aymkdn/ExcelPlus#8]
Hi
As your code depends on SheetJS/js-xlsx, imbedded in a homemade sheetjs.all.min.js library, how do you deal with js-xlsx updates ? If one decide to use ExcelPlus, how can he be sure to get the latest js-xlsx updates ?
Would it be possible to use the "official" js-xlsx library instead of sheetjs.all.min.js ?

best regards

**Comment by Aymkdn** [comment]
Hi,

When I did ExcelPlus the js-xlsx project was dead. I'm happy to see that after 2 years they have finally reopened it :-)

My version of js-xlsx has a few changes:

1. IE8 compatibility (https://github.com/SheetJS/js-xlsx/pull/351 and https://github.com/SheetJS/js-xlsx/issues/350)
2. IE10 compatibility (https://github.com/SheetJS/js-xlsx/pull/349)
3. I provide the `fmtid` in order to detecting the date cells (https://github.com/SheetJS/js-xlsx/pull/349)

I'll have to test the last release to see if the Date cell detection now works. Also I think the IE10 compatibility is a good thing to have....

So, to summarize, if you want to use the js-xslx 0.8.3, and make it work with ExcelPlus, you probably only need to change https://github.com/Aymkdn/ExcelPlus/blob/master/2.4.1/excelplus-2.4.1.js#L418 ... as the other stuff is for IE compatibility.

**Comment by Aymkdn** [comment]
FYI I'll be on vacation for two weeks. On my return I'll try to take some time to test the new version of js-xlsx and see how to use the lastest releases.

**Comment by lbayle** [comment]
No hurry, I haven't done my choice anyway. It was more like an advice because dependencies are a pain for developers & lib users. Maybe the best way is to contribute your work to the original library.

best regards

**Comment by Aymkdn** [comment]
FYI two of my pull requests have been accepted by the owners (support for IE8).

I've submitted another request to correctly detect the date cells (https://github.com/SheetJS/js-xlsx/pull/581).... this one is **critical** as the current js-xlsx version doesn't support the date cells.

If this PR is accepted, then ExcelPlus will be compatible with the last version of js-xlsx.

**Comment by lbayle** [comment]
Thank you

**Comment by Aymkdn** [comment]
Fixed into the new release (v2.5)
